recipes/hklpy2/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "hklpy2" %}
+{% set version = "0.0.23" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hklpy2-{{ version }}.tar.gz
+  sha256: 520fadf2a358feead3c72fd9fc675039159a3b6cfc79e2475604faec5640bb6a
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  entry_points:
+    - no_op = hklpy2.backends.no_op:NoOpSolver
+    - hkl_soleil = hklpy2.backends.hkl_soleil:HklSolver
+    - th_tth = hklpy2.backends.th_tth_q:ThTthSolver
+  noarch: python
+
+requirements:
+  host:
+    - "python >=3.10"
+    - setuptools-scm >=8
+    - setuptools >=64.0
+    - pip
+  run:
+    - "python >=3.10"
+    - hkl
+    - numpy
+    - ophyd
+    - packaging
+    - pint
+
+test:
+  imports:
+    - hklpy2
+  commands:
+    - pip check
```

Comment:
You may want to add tests for the entrypoints here, something like outputting the help would do.

Reply:
Since there is no command-line entry point (this package provides a library, not an application), there is no command-line help.  The entry-points are reported by this test command: https://github.com/conda-forge/staged-recipes/blob/84f167d8787fec83e751da040c19e0e789f9fc5c/recipes/hklpy2/meta.yaml#L40

This test command could be expanded by testing for the number of entry-points expected.

```bash
python -c "import hklpy2; assert len(hklpy2.solvers())==3; print(hklpy2.solvers())"
```